src/theme/book.js

```diff
@@ -6,12 +6,19 @@ window.onunload = function () { };
 // Global variable, shared between modules
 function playpen_text(playpen) {
     let code_block = playpen.querySelector("code");
+    let regex = /^\$(?: )/;
 
     if (window.ace && code_block.classList.contains("editable")) {
         let editor = window.ace.edit(code_block);
         return editor.getValue();
     } else {
-        return code_block.textContent;
+        // It is common for code blocks to contain $ in the text. If they do, clipboard will copy that symbol and the ensuing paste into terminal will fail
+        if (regex.test(code_block.textContent)) {
```

Comment:
Is this `if` necessary? If we do a replace and nothing matches we are going to retrieve the input string at the output, no?